frontend/device/pocketbook/device.lua

```diff
@@ -17,6 +17,29 @@ _G.POCKETBOOK_FFI = true
 local function yes() return true end
 local function no() return false end
 
+local browser_app ="/ebrmain/bin/webbrowser.sh"
+
+local function isUrl(s)
+    return type(s) == "string" and s:match("*?://")
+end
+
+local function isCommand(s)
+    return os.execute("which "..s.." >/dev/null 2>&1") == 0
+end
+
+local function runCommand(command)
+      return os.execute(command) == 0
+end
```

Comment:
I suspect this is all duplicated, which means it probably wants to be refactored into util or something.